apm-sniffer/apm-toolkit-activation/apm-toolkit-micrometer-activation/src/main/java/org/apache/skywalking/apm/toolkit/activation/micrometer/MicrometerReceiverTracingHandlerInterceptor.java

```diff
@@ -60,7 +60,6 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
             SpanLayer spanLayer = TaggingHelper.toLayer(context.getAllKeyValues());
             AbstractSpan abstractSpan = ContextManager.activeSpan();
             abstractSpan
-                .setPeer(SpanHelper.tryToGetPeer(context.getRemoteServiceAddress(), context.getAllKeyValues()))
```

Comment:
We don't need peer on the receiver side?

Reply:
It is optional in most cases, and required in MQ consumer and producer cases.

Reply:
@wapkch I think this should be kept. Why do you remove this?

Reply:
Because it breaks the Test / resttemplate-6.x case on entry span, which expects an empty peer, but the peer from `tryToGetPeer` here will be `/resttemplate-6.x-scenario/resttemplate/syncback` now, it seems meaningless in this case. And now i realize that the peer is required in MQ consumer and producer, but i'm not sure if i should keep this and modify the peer of entry span in Test / resttemplate-6.x case to `/resttemplate-6.x-scenario/resttemplate/syncback`?

Reply:
Why the peer is a URI? It should be IP and port only. Peer should be a network socket address only.

Reply:
Because IP and port can not retrieved from Micrometer ObservationContext in some cases, but a URI can be easily found from the keyValues in it. So i think we may should distinguish receiver side to MQ receiver and other receivers because MQ receiver need a peer and others do not. cc @marcingrzejszczak

Reply:
If you use URI, the IP:PORT mapping data in the backend would easily burn out and crash the OAP due to OOM.
[STAM paper](https://skywalking.apache.org/docs/main/next/en/papers/stam/) introduced why this is required to build topology in pure streaming rather than a time window oriented.

Reply:
I see. I will try to find another way.

Reply:
I add back peer on the receiver side and remove the uri path logic in tryToGetPee method.